# Tutorial 2: MCGAE's spatial clustering of colorectal cancer liver metastasis data elucidated the phenomenon of tumor invasion

## The main references of the article
1.	Rao A, Barkley D, Franca GS et al. Exploring tissue architecture using spatial transcriptomics, Nature 2021;596:211-220.
2.	Armingol E, Officer A, Harismendy O et al. Deciphering cell-cell interactions and communication from gene expression, Nat Rev Genet 2021;22:71-88.
3.	Ji N, van Oudenaarden A. Single molecule fluorescent in situ hybridization (smFISH) of C. elegans worms and embryos, WormBook 2012:1-16.
4.	Chen KH, Boettiger AN, Moffitt JR et al. RNA imaging. Spatially resolved, highly multiplexed RNA profiling in single cells, Science 2015;348:aaa6090.
5.	Moffitt JR, Bambah-Mukku D, Eichhorn SW et al. Molecular, spatial, and functional single-cell profiling of the hypothalamic preoptic region, Science 2018;362.
6.	Lubeck E, Coskun AF, Zhiyentayev T et al. Single-cell in situ RNA profiling by sequential hybridization, Nat Methods 2014;11:360-361.
7.	Lee JH, Daugharthy ER, Scheiman J et al. Highly multiplexed subcellular RNA sequencing in situ, Science 2014;343:1360-1363.
8.	Wang X, Allen WE, Wright MA et al. Three-dimensional intact-tissue sequencing of single-cell transcriptional states, Science 2018;361.
9.	Ji AL, Rubin AJ, Thrane K et al. Multimodal Analysis of Composition and Spatial Architecture in Human Squamous Cell Carcinoma, Cell 2020;182:497-514.e422.
10.	Rodriques SG, Stickels RR, Goeva A et al. Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution, Science 2019;363:1463-1467.
11.	Chen A, Liao S, Cheng M et al. Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays, Cell 2022;185:1777-1792.e1721.
12.	Moncada R, Barkley D, Wagner F et al. Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas, Nat Biotechnol 2020;38:333-342.
13.	Satija R, Farrell JA, Gennert D et al. Spatial reconstruction of single-cell gene expression data, Nat Biotechnol 2015;33:495-502.
14.	Dries R, Zhu Q, Dong R et al. Giotto: a toolbox for integrative analysis and visualization of spatial expression data, Genome Biol 2021;22:78.
15.	Zhao E, Stone MR, Ren X et al. Spatial transcriptomics at subspot resolution with BayesSpace, Nat Biotechnol 2021;39:1375-1384.
16.	Long Y, Ang KS, Li M et al. Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST, Nat Commun 2023;14:1155.
17.	Dong K, Zhang S. Deciphering spatial domains from spatially resolved transcriptomics with an adaptive graph attention auto-encoder, Nat Commun 2022;13:1739.
18.	Ren H, Walker BL, Cang Z et al. Identifying multicellular spatiotemporal organization of cells with SpaceFlow, Nat Commun 2022;13:4076.
19.	Shi X, Zhu J, Long Y et al. Identifying spatial domains of spatially resolved transcriptomics via multi-view graph convolutional networks, Brief Bioinform 2023;24.
20.	Hu J, Li X, Coleman K et al. SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network, Nat Methods 2021;18:1342-1351.
21.	Xu C, Jin X, Wei S et al. DeepST: identifying spatial domains in spatial transcriptomics by deep learning, Nucleic Acids Res 2022;50:e131.
22.	Zong Y, Yu T, Wang X et al. conST: an interpretable multi-modal contrastive learning framework for spatial transcriptomics, bioRxiv 2022:2022.2001.2014.476408.
23.	Wang B, Luo J, Liu Y et al. Spatial-MGCN: a novel multi-view graph convolutional network for identifying spatial domains with attention mechanism, Briefings in Bioinformatics 2023;24:bbad262.
24.	He K, Zhang X, Ren S et al. Deep Residual Learning for Image Recognition. In: 2016 IEEE Conference on Computer Vision and Pattern Recognition (CVPR). 2016, p. 770-778.
25.	Zhang C, Yang Y, Tang S et al. Contrastively generative self-expression model for single-cell and spatial multimodal data, Briefings in Bioinformatics 2023;24:bbad265.
26.	Kipf TN, Welling M. Semi-Supervised Classification with Graph Convolutional Networks. 2016, arXiv:1609.02907.
27.	Veličković P, Fedus W, Hamilton WL et al. Deep Graph Infomax. 2018, arXiv:1809.10341.
28.	Paszke A, Gross S, Massa F et al. PyTorch: an imperative style, high-performance deep learning library.  Proceedings of the 33rd International Conference on Neural Information Processing Systems. Curran Associates Inc., 2019, Article 721.
29.	Kingma DP, Ba J. Adam: A method for stochastic optimization, arXiv preprint arXiv:1412.6980 2014.
30.	Paszke A, Gross S, Massa F et al. Pytorch: An imperative style, high-performance deep learning library, Advances in neural information processing systems 2019;32.
31.	Maynard KR, Collado-Torres L, Weber LM et al. Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex, Nat Neurosci 2021;24:425-436.
32.	Li H, Calder CA, Cressie N. Beyond Moran's I: Testing for Spatial Dependence Based on the Spatial Autoregressive Model, Geographical Analysis 2007;39:357-375.
33.	Wu Y, Yang S, Ma J et al. Spatiotemporal Immune Landscape of Colorectal Cancer Liver Metastasis at Single-Cell Level, Cancer Discovery 2022;12:134-153.
